code/game/machinery/computer/robot.dm

```diff
@@ -274,7 +274,7 @@
 		A << 'sound/machines/warning-buzzer.ogg'
 	for(var/mob/living/silicon/robot/R in cyborg_list)
 		if(!R.scrambledcodes && !isMoMMI(R))
-			to_chat(R, "<span style=\"font-family:Courier\"><b>\[<span class='danger'>ALERT</span>\] Emergency Self-Destruct sequence initiated. This unit will self-destruct in [formatTimeDuration(cyborg_detonation_time-world.time)] unless a termination signal is recieved.</b></span>")
+			to_chat(R, "<span style=\"font-family:Courier\"><b>\[<span class='danger'>ALERT</span>\] Emergency Self-Destruct Sequence Initiated. This unit will self-destruct in [formatTimeDuration(cyborg_detonation_time-world.time)] unless a termination signal is received.</b></span>")
```

Comment:
no need to capitalize Sequence Initiated, the first three words are capitalized because they are A Sci-Fi Thing™.

Reply:
eh?

Reply:
what do you mean by "A Sci-Fi Thing"

Reply:
He means that he perceives "Emergency Self-Destruct" as a proper noun, the name of the sequence. Like saying "Clausewitz sequence initiated". Sequence could easily be included or not included in the name, but initiated seems unnecessary to capitalize.

Reply:
Yeah but the AI message just above is likewise capitalized 
![image](https://user-images.githubusercontent.com/7573912/130143951-e06615b7-2802-43b3-b88c-1077f3e048e0.png)

I'm a sucker for consistency and I sorta like that one better.

Reply:
/shrug
go for it